t/01-archive_folder.t

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/perl
+# This program is open source, licensed under the PostgreSQL License.
+# For license terms, see the LICENSE file.
+#
+# Copyright (C) 2012-2021: Open PostgreSQL Monitoring Development Group
+
+use strict;
+use warnings;
+
+use lib 't/lib';
+use pgNode;
+use Test::More tests => 35;
+
+my $node        = pgNode->new('prod'); # declare instance named "prod"
+my $archive_dir = $node->archive_dir;
+my $wal;
+my $time;
+
+# create the instance and start it
+$node->init(has_archiving => 1, allows_streaming => 1);
+$node->start;
+
+# generate three archives
+# split create table and insert to produce more data in WAL
+$node->psql('template1', 'create table t (i int primary key)');
+$node->psql('template1', 'insert into t select generate_series(1,10000) i');
+$node->switch_wal;
+$node->psql('template1', 'insert into t select generate_series(10001,20000) i');
+$node->switch_wal;
+$node->psql('template1', 'insert into t select generate_series(20001,30000) i');
+$wal = $node->switch_wal;
+$node->wait_for_archive($wal);
+
+### Begin of tests ###
+
+# simple success check
+$node->command_checks_all( [
+    './check_pgactivity', '--service'  => 'archive_folder',
```

Comment:
Note that this requires the script to be executable, which I think is not the case right now.  Is it?

I'm in favor of changing the mode bits rather than putting `perl ./check_pgactivity`  everywhere.

Reply:
This has been committed in a1dfb35fce499e9147df50bc56d1c564caf7b2de